apps/api-example/schema-v2.graphql

```diff
@@ -46,6 +51,43 @@ A date-time string at UTC, such as 2019-12-03T09:54:33Z, compliant with the date
 """
 scalar DateTime
 
+type Event {
+  createdAt: DateTime!
+  description: JSON!
```

Comment:
Should be `Node!` from slate.

Reply:
are you able to point me where should I change the code and how? I am actually not sure 🤔 I know that this `description: JSON!` is autogenerated but I am not entirely sure from where exactly, and what should be the corrected source value